AmplifyTests/CategoryTests/Auth/AuthCategoryConfigurationTests.swift

```diff
@@ -0,0 +1,367 @@
+//
```

Comment:
this file seems to be in. "tests" folder -- is this where the unit tests live or is there a specific unit tests folder?

Reply:
Hmm not sure what i was thinking when i put this comment down.. resolving.